### PR TITLE
Opt-751 tampermonkey causes healthie calendar to not be pageable

### PR DESCRIPTION
**Pull request by tonyebrown**
# Pull Request Checklist

- [x] Have you named your branch to include the Jira ticket number?
- [x] Have you added a label to this PR? (enhancement, bug, chore, tech debt, deployment)

_Reference Document for PR standards: https://vorihealth.atlassian.net/l/cp/AEP3QU1P_

### CODE REVIEW

**Review by dheatherington — APPROVED** [review]

File: careplan.js [hunk 1]
```diff
@@ -15,7 +15,7 @@
 /* globals contentful */
 
 //Enable/Disable debug mode
-let debug = false;
+let debug = true;
```
Comment (dheatherington) [inline careplan.js]: Let's make sure to change this to false before merging this

File: careplan.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 // ==UserScript==
-// @name         Healthie Care Plan Integration
+// @name         Healthie Care Plan Integration dev
```
Comment (dheatherington) [inline careplan.js]: Let's keep the script name the same

File: careplan.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
...
 // @namespace    http://tampermonkey.net/
 // @version      0.60
```
Comment (dheatherington) [inline careplan.js]: Increment version to prompt script update

File: careplan.js (hunk 2 above)
Comment (tonyebrown) [reply careplan.js]: oops good catch

**Review by dheatherington — APPROVED** [review]

**Pull request merged**